Plugins/DotNET/DotNET.cpp

```diff
@@ -81,6 +75,38 @@ bool DotNET::InitThunks()
         }
     }
 
+    if (!nethost)
+    {
+        const auto hostBaseDir = "/usr/share/dotnet/packs/Microsoft.NETCore.App.Host.linux-x64/";
+        const auto hostLibSuffix = "/runtimes/linux-x64/native/libnethost.so";
+
+        DIR* dir = opendir(hostBaseDir);
+
+        if (dir != nullptr)
+        {
+            dirent* directoryEntry = readdir(dir);
+
+            while (directoryEntry != nullptr)
```

Comment:
Do you always have exactly one path installed? If you have multiple versions, this will pick one semi-randomly.
I'm not sure this is even an issue - I half expect all the versions of libnethost.so to be byte-per-byte identical, but it's something we need to look into before merging.

Reply:
I think this is currently an issue with docker as the image targets a major version number (dotnet-3.1 currently), and this tag is updated with a minor version release (3.1.3 -> 3.1.4) essentially breaking dotnet per release (or requiring a manual config update) if you are using docker.

Reply:
Correct, the dotnet version needs to be updated in the nwnxee dockerfile as well. 
There should only be one libnethost.so per dotnet install (I would hope.)

With this fix in place we should be able to increment the dotnet version in the dockerfile and not have to worry adding paths anymore. 

I have not tested this pr, but if it works it would make our lives easier.

Reply:
I have merged #905 for now until I (or someone else) verify that this will work as expected if you have multiple versions of dotnet installed. Otherwise might need to sort the dentrys and go from the top.